daprdocs/content/en/reference/components-reference/supported-secret-stores/hashicorp-vault.md

```diff
@@ -79,6 +79,37 @@ For Kubernetes, you can use the Helm Chart: <https://github.com/hashicorp/vault-
 {{% /codetab %}}
 
 {{< /tabs >}}
+
+
+## Multiple Key-Values per Secret
+
+HashiCorp Vault supports multiple key-values in a secret. While this behavior is ultimately dependent on the underlying [secret engine](https://www.vaultproject.io/docs/secrets#secrets-engines) configured by `enginePath`, it is something that might change the way you store and retrieve keys from Vault. For instance, multiple key-values in a secret is the behavior exposed the `secret` engine, the default engined configured by the `enginePath` field.
+
+When retrieving secrets, a JSON payload is returned with the key names as fields and their respective values.
+
+Suppose you add a secret to your Vault setup as follows:
+
+```shell
+vault kv put secret/dapr/mysecret firstKey=aValue secondKey=anotherValue thirdKey=yetAnotherDistinctValue
```

Comment:
Curious if I should make a mention to the `vaultKVPrefix` which defaults to `dapr` (see line 64 above) here.

I know I, when testind dapr before, would have appreciated this as I was a bit surprised at first about this "dapr" prefix.

Reply:
Yes. Always added useful information like this.

Reply:
Added a short sentence refering to the vaultKVPrefix